pkg/drivers/k8scel/transform/cel_snippets.go

```diff
@@ -82,6 +84,49 @@ const (
 		)
 	)
 	`
+
+	// Expression to exclude objects in globally excluded namespaces and namespace objects themselves if they're in the exclusion list.
+	matchGlobalExcludedNamespacesGlob = `
```

Comment:
can you add a comment that this is for config exclusion?

Reply:
updated comment for this.